Snippets/backstat.sqlpromptsnippet

```diff
@@ -0,0 +1,13 @@
+    SELECT  r.session_id as spid, command, start_time, percent_complete,
```

Comment:
-1) Format was to make code readable...
0) Naming: yea, backstat because it is for backup-status (and restore status, but backretstat got weird :))
1) I can make that change...
2) I'll do a SQL Prompt KY on it after updates
3) Coding style of mine--hold habit from Lisp days carried forward to having had an OR not wrapped in parenthesis wreak havoc on an SP...
4) For GitHub, but l'll be keeping it in my copy...